notifirehq/notifire
# feat: Change prettier config to js so it can be 'extended'

### What changed? Why was the change needed?


- In the Design System package setup ([NV-3691](https://linear.app/novu/issue/NV-3691/%E2%9A%A1-package-setup-to-improve-dx)), we will add a Panda-specific Prettier plugin to keep our styles consistent!
- Our current `.prettierrc` can't be extended, so in order to enable "[extending](https://prettier.io/docs/en/configuration.html#:~:text=Note%3A%20This%20method%20does%20not%20offer%20a%20way%20to%20extend%20the%20configuration%20to%20overwrite%20some%20properties%20from%20the%20shared%20configuration.%20If%20you%20need%20to%20do%20that%2C%20import%20the%20file%20in%20a%20.prettierrc.js%20file%20and%20export%20the%20modifications%2C%20e.g%3A)" it, we must switch to a JS file

### Questions

- [ ] Are there any other places we explicitly refer to `.prettierrc`?

### Validation Steps

1. Check-out the branch
2. Edit a TypeScript file (change a string to double-quotes or mess with formatting)
3. Save
4. The file should reformat properly
